components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationUserResidentResolverServiceImpl.java

```diff
@@ -225,6 +226,25 @@ the accessed organization (lower level)
         return basicOrganizationList;
     }
 
+    public User getUserFromResidentOrgId(String userId, String userResidentOrganizationId)
+            throws OrganizationManagementException {
+
+        User user = null;
+        try {
+            String associatedTenantDomainForOrg = resolveTenantDomainForOrg(userResidentOrganizationId);
```

Comment:
Can `userResidentOrganizationId` be null?

Reply:
The method invocator can pass null, but I hope you meant `associatedTenantDomainForOrg` can be null?. In such case I also feel it won't be null

Reply:
No. It can not be null.

Reply:
So we may not need the null check